src/core/client/stream/containers/UserBoxContainer.tsx

```diff
@@ -31,13 +46,19 @@ export class UserBoxContainer extends Component<InnerProps> {
       local: {
         authPopup: { open, focus, view },
       },
+      user,
     } = this.props;
+
+    if (user) {
+      return <UserBoxAuthenticatedContainer user={user} />;
```

Comment:
Do we need an extra container `UserBoxAuthenticatedContainer`? Wouldn't it make sense to have the logic in this file?